packages/common/src/projects/_internal/ProjectUiSchemaEdit.ts

```diff
@@ -58,6 +58,22 @@ export const uiSchema: IUiSchema = {
             },
           },
         },
+        {
+          labelKey: "{{i18nScope}}.fields.combobox.label",
+          scope: "/properties/view/properties/combobox",
```

Comment:
per the comment above, this will become `/properties/tags` rather than being nested under `/properties/view`